lib/utils/oclif-utils.ts

```diff
@@ -69,3 +69,28 @@ export function capitanoizeOclifUsage(
 		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, (match) => `<${match}>`)
 		.toLowerCase();
 }
+
+export async function getCommandsFromManifest() {
+	const fs = await import('fs');
+	let manifest;
+	try {
+		manifest = JSON.parse(
+			await fs.promises.readFile('./oclif.manifest.json', 'utf8'),
+		);
+	} catch (e) {
+		if (e.code === 'ENOENT') {
+			throw new Error('Oclif manifest not found.');
+		} else {
+			throw e;
+		}
+	}
+	return manifest.commands;
+}
+
+export async function getCommandIdsFromManifest() {
+	const commands = await getCommandsFromManifest();
+	if (commands == null) {
+		throw new Error('Commands second not found in manifest.');
```

Comment:
What does _"Commands second"_ mean? Would this error be raised if a developer just added a new oclif command and ran the CLI before running `npm run build` (i.e. the manifest hasn't been updated for the new command). Maybe this error message could also suggest the fix:
```
throw new Error(stripIndent`
    Commands second not found in ${manifestPath}.
    If you are developing the CLI, run \`npm run build\` to fix it.`);
```

Reply:
Ah, that should be `section`.   I don't think that scenario would cause it though.  It would only throw if the manifest file was present, and valid JSON, but just missing that section.  I'm not sure how that might happen, but I'm feeling a bit cautious relying on this manifest, so want to cover all bases.